### PR TITLE
more detail to permission request edit page

### PR DESCRIPTION
**Pull request by herb**

### CODE REVIEW

**Comment by herb** [comment]
before:
screenshot 2016-02-24 14 04 23

after:
screenshot 2016-02-24 14 04 00

File: grouper/fe/templates/permission-request-update.html [hunk 1]
```diff
@@ -30,8 +30,12 @@
                 </tr>
                 <tr>
                     <td colspan="5">
-                        <span style="padding-left: 10px;">
-                            <strong>Reason:</strong> <em>{{comment.comment}}</em>
+                        <dl class="dl-horizontal" style="margin-bottom: 0px;">
+                            <dt>Permission:<dt>
+                            <dd><a href="/permissions/{{ request.permission.name }}">
+                                {{ request.permission.name }}</a></dd>
```
Comment (lfaraone) [inline grouper/fe/templates/permission-request-update.html]: We don't have a linter for our templates, but I think ideal style is to put closing tags on a separate line from content if the opening tags were.

File: grouper/fe/templates/permission-request-update.html [hunk 2]
```diff
@@ -30,8 +30,12 @@
                 </tr>
                 <tr>
                     <td colspan="5">
-                        <span style="padding-left: 10px;">
-                            <strong>Reason:</strong> <em>{{comment.comment}}</em>
+                        <dl class="dl-horizontal" style="margin-bottom: 0px;">
```
Comment (lfaraone) [inline grouper/fe/templates/permission-request-update.html]: This isn't tabular data, so it shouldn't be nested in a table. Using tables for layout is bad news bears.

File: grouper/fe/templates/permission-request-update.html (hunk 1 above)
Comment (herb) [inline grouper/fe/templates/permission-request-update.html]: fixed

File: grouper/fe/templates/permission-request-update.html (hunk 2 above)
Comment (herb) [inline grouper/fe/templates/permission-request-update.html]: i copied/pasted from the join request flow. so yeah, probably a TODO for cleanup.

**Pull request merged**